src/fake-timers-src.js

```diff
@@ -33,13 +33,37 @@ var globalObject = require("@sinonjs/commons").global;
  * @property {() => Promise<number>} runToLastAsync
  * @property {() => void} reset
  * @property {(systemTime: number | Date) => void} setSystemTime
- * @property {any} performance
+ * @property {({now(): number})} performance
  * @property {(prev: any) => number[]} hrTime
  * @property {() => void} uninstall Uninstall the clock.
  * @property {any} methods
  */
 
-// eslint-disable-next-line complexity
+/**
+ * Configuration object for the `install` method.
+ *
+ * @typedef {object} Config
+ * @property {number|Date} now a number (in milliseconds) or a Date object (default epoch)
+ * @property {string[]} toFake names of the methods that should be faked.
+ * @property {number} loopLimit the maximum number of timers that will be run when calling runAll()
+ * @property {Boolean} shouldAdvanceTime tells FakeTimers to increment mocked time automatically (default false)
+ * @property {Number} advanceTimeDelta increment mocked time every <<advanceTimeDelta>> ms (default: 20ms)
```

Comment:
shouldn't this be `number` rather than `Number` (i.e. asking for a primitive number rather than a boxed Number object) ? Same for `boolean` on the previous line.

The typescription documentation about writing types says that you almost never want to use the boxed types in type declarations.

Reply:
Pretty sure we want to use primitives rather than boxed types everywhere. That also aligns with how I read [the JSDoc docs](https://jsdoc.app/tags-type.html)

Reply:
Yep it's just a typo. Have spent too much time writing the constructors recently in other projects and had it stuck in my head 😂